tests/unit/docker/test_hub_build.py

```diff
@@ -91,6 +91,23 @@ def test_hub_build_uses():
     HubIO(args).build()
 
 
+@pytest.mark.timeout(360)
+def test_hub_build_uses():
+    args = set_hub_build_parser().parse_args(
+        [os.path.join(cur_dir, 'hub-mwu'), '--test-uses', '--raise-error']
+    )
+    assert HubIO(args).build()["is_build_success"]
```

Comment:
please stick to single quote

Reply:
https://github.com/jina-ai/jina/pull/1958/commits/7301267d0a05735aa9a251c3f25dab8b1bb3d9be